service.go

```diff
@@ -218,6 +218,21 @@ func (self *Service) GetConfig() (*ServiceConfig, error) {
 	}, nil
 }
 
+func (self *Service) GetDelayedAutoStartInfo() (bool, error) {
```

Comment:
Still need to rename this to `GetDelayedAutoStart`.

Reply:
Right! Done.